x-pack/plugins/enterprise_search/server/utils/create_ml_inference_pipeline.test.ts

```diff
@@ -0,0 +1,187 @@
+/*
```

Comment:
Suggestion: One for tech debt - i would move this out of utils and into the lib/ml folder.

Reply:
Filed https://github.com/elastic/enterprise-search-team/issues/2752 for this.